packages/entities/entities-upstreams-targets/src/components/UpstreamsFormActiveHealthCheck.cy.ts

```diff
@@ -280,7 +280,7 @@ describe('<UpstreamsFormActiveHealthCheck/>', { viewportHeight: 700, viewportWid
     })
   })
 
-  // TODO: remove when api will support `headers` property
+  // Konnect supports `headers` property now
```

Comment:
I think we can merge this test with the following one:
1. Remove line 283 to line 295
2. Change line 296 to `it('Should bind headers data correctly', () => {`
3. Remove line 301
4. Then remove line 2